src/streamlink/plugins/artetv.py

```diff
@@ -9,24 +9,29 @@
 from streamlink.stream import HLSStream
 
 log = logging.getLogger(__name__)
-JSON_VOD_URL = "https://api.arte.tv/api/player/v1/config/{0}/{1}?platform=ARTE_NEXT"
-JSON_LIVE_URL = "https://api.arte.tv/api/player/v1/livestream/{0}"
+JSON_VOD_URL = "https://api.arte.tv/api/player/v2/config/{0}/{1}"
+JSON_LIVE_URL = "https://api.arte.tv/api/player/v2/config/{0}/LIVE"
+V2_API_BEARER = "MzYyZDYyYmM1Y2Q3ZWRlZWFjMmIyZjZjNTRiMGY4MzY4NzBhOWQ5YjE4MGQ1NGFiODJmOTFlZDQwN2FkOTZjMQ"
```

Comment:
Please move the constants from the global scope to the class as class attributes